src/Dialogs/Settings/Panels/SymbolsConfigPanel.cpp

```diff
@@ -135,6 +136,13 @@ static constexpr StaticEnumChoice wind_arrow_list[] = {
   { 0 }
 };
 
+static constexpr StaticEnumChoice skylines_traffic_list[] = {
```

Comment:
This option is badly named and badly described. This is not about whether/how to display names on the map; that's just one of three options. It can also be used to disable the traffic display completely.

Reply:
You are right, I reworked the code with the skylines_traffic_map_mode.